_docs/services/relational-database.md

```diff
@@ -13,35 +13,33 @@ If your application uses relational databases for storage, you can use the AWS R
 
 ## Plans
 
-Plan Name                | Description                                                                  | Software Version |
----------                | -----------                                                                  | ---------------- |
-`shared-psql`            | Shared PostgreSQL database for prototyping (no sensitive or production data) | 9.5.15           |
```

Comment:
Where did shared go? are we no longer allowing it?

Reply:
Nothing has been formalized yet, but the plan is to phase them out.  I figured this would be a first step in steering folks away from them.  Do you feel we should hold off on removing them from the documentation until that's ironed out some more?

If so, then I'd like to just verify the version number of each in production as a part of the update, because I think the shared DB instances have been updated in recent months.

Reply:
if we're going to phase them out, we should do so loudly rather than quietly
If we're doing this simply to discourage their use, then I have mixed feelings about this


Maybe leave them in and have a larger discussion tomorrow after standup?

Reply:
Sounds good. :-)

Reply:
Come to think of it, this wouldn't follow our deprecation policy anyway, so I'll add these back in for now.